db/migrate/20160123063553_add_turn_to_games.rb

```diff
@@ -0,0 +1,5 @@
+class AddTurnToGames < ActiveRecord::Migration
+  def change
+    add_column :games, :turn, :integer
```

Comment:
Another way of doing this would be.

Maybe

`active_player_id`

Then could setup the relationship in the same way we did with the black/white player ids.

`belongs_to :active_player, class_name: 'User'`

Then we could do things like. 
`game.active_player = black_player`

Or 

```
unless game.active_player == current_user
  # return unauthorized (example)
end
```